src/hu_nmt/data_augmentator/dependency_parsers/stanza_dependency_parser.py

```diff
@@ -41,7 +41,8 @@ def sentence_to_node_relationship_list(nlp_pipeline, sent: str) -> List[NodeRela
             token = word.to_dict()
             target_key = f'{token["text"].lower()}_{token["id"]}'
             target_postag = token['upos']
-            target_lemma = token['lemma']
+            #target_lemma = token['lemma']
+            target_lemma = token.get('lemma')
```

Comment:
why did we need to change this to `.get()` and not for the other attributes? Do we not always have the lemma there?

Reply:
yes, I got errors earlier as there was no lemma, should I change the rest to be sure?

Reply:
I assumed we got all of these correctly, so I'm a bit surprised we don't always get the lemma. I think for the lemma it's fine, because we are not using it for anything rn if I'm correct. What we could do is to rewrite the lemma to use `get` and if there's no lemma, just return the token itself as the lemma (and still populate the lemma field). For other fields I think it can stay like this, we need to make sure we always get a POS tag and a deprel.